proxy/envoy/testdata/cds-ssl-context.json.golden

```diff
@@ -9,8 +9,7 @@
     "ssl_context": {
      "cert_chain_file": "/etc/certs/cert-chain.pem",
      "private_key_file": "/etc/certs/key.pem",
-     "ca_cert_file": "/etc/certs/root-cert.pem",
-     "verify_subject_alt_name": []
```

Comment:
@myidpt does this change semantics of the cluster? If I remember we want this to be an empty array rather than dropped since the two mean different things: missing implies all allowed, empty implies none allowed. We might need to define another schema resource to deal with this problem.